### PR TITLE
Improve readme section about local development

### PR DESCRIPTION
**Pull request by silversonicaxel**
I am interested in running locally the project.
In order to do that I was trying to follow the README section.
Since it is pretty "poor" I thought to improve it.

Current work in progress is intended to do that.

But I need some support from @elamperti 

Can we try to fix it together?

I am trying to add all variables information and all commands to execute in order to have a running project.

I am at the stage that, react application is up and running, what is not working is the fetching of the lastfm user information

Screenshot 2024-06-28 at 10 52 10
 
and what seems to miss here is a php server to do that, and indeed a configuration for `SQLITE_DB_PATH` variable. Can you support on that?

### CODE REVIEW

**Comment by elamperti** [comment]
Hi, thanks for your interest in improving the docs, I appreciate it! 

From what I see in your changes, you are trying to start the docker service directly using docker compose, but the whole project should start correctly with `yarn start` (which starts both the webserver and the docker service).

If the error persists using this method, check the network tab to see what happens with the calls to `user.php`, there should be enough information about what's failing.  Let me know if you need more help!

**Comment by silversonicaxel** [comment]
I fixed the commands section and

```
<br />
<b>Warning</b>:  SQLite3::prepare(): Unable to prepare statement: 1, no such table: settings in <b>/var/www/html/api/v2/inc/database.php</b> on line <b>21</b><br />
<br />
<b>Fatal error</b>:  Uncaught Error: Call to a member function bindValue() on bool in /var/www/html/api/v2/inc/database.php:22
Stack trace:
#0 /var/www/html/api/v2/user.php(45): Database-&gt;get_settings('silversonicaxel')
#1 {main}
  thrown in <b>/var/www/html/api/v2/inc/database.php</b> on line <b>22</b><br />
```

this is the error I see.

**Comment by elamperti** [comment]
Considering it's complaining about the `settings` table being missing, there's probably some issue running the db migrations (not sure why). You can try removing the `database` directory in the repo, which contains the development db, and starting the project again (ideally using `yarn start`) should run the migration script when the `sqlite-web` container boots up.

**Comment by silversonicaxel** [comment]
Screenshot 2024-06-30 at 10 09 24

`settings` table is there, but for some reason, I do not have any `database` folder in my repository. 
might it be the issue?

File: README.md [hunk 1]
```diff
@@ -5,39 +5,58 @@
...
 ### Translations
+
 To join the translators team, follow the link at the bottom of the language selector in the application ("Want to add your language?").
 
 ## Development setup
-You'll need [a Last.fm API account](https://www.last.fm/api/account/create)! (and probably a Discogs API key as well)
+
+You need a [Last.fm API account](https://www.last.fm/api/account/create) to be able to interact with OpenWebScrobbler and query Last.fm database.
+
+You need a [Discogs API application](https://www.discogs.com/settings/developers) to be able to query Discogs database.
```
Comment (elamperti) [inline README.md:31]: Possible simplification/improvement for these lines and the list below:

```suggestion

  - Clone the repository
  - Copy `.env` to `.env.development.local`, then set at least  the required variables
    - You'll need a [Last.fm API account](https://www.last.fm/api/account/create) to be able to interact with Open Scrobbler (it's used for authentication and queries). Once you have your keys, fill in `REACT_APP_LASTFM_API_KEY` and `LASTFM_API_KEY` (same value in both) and `LASTFM_SECRET`

    - Optional:  to interact with Discogs, create a [Discogs application](https://www.discogs.com/settings/developers) to get API keys and fill in `DISCOGS_API_KEY` and `DISCOGS_SECRET`
```

I'd also add two comments in `.env` specifying required/optional sections:

```
# Required
REACT_APP_HOST=localhost:3000
REACT_APP_LASTFM_API_KEY=
LASTFM_API_KEY=
LASTFM_SECRET=
SQLITE_DB_PATH=/var/www/html/database/development.db

# Optional
DISCOGS_API_KEY=
DISCOGS_SECRET=
REACT_APP_ANALYTICS_MEASUREMENT_ID=
REACT_APP_GROWTHBOOK_API_KEY=
REACT_APP_LINK_TO_TRANSLATIONS="https://github.com/elamperti/OpenWebScrobbler/blob/main/CONTRIBUTING.md#translations"
REACT_APP_SENTRY_DSN=

# Only for development: uncomment and set these numbers to your user/group IDs
# APACHE_RUN_USER=#1000
# APACHE_RUN_GROUP=#1000
```

**Comment by silversonicaxel** [comment]
Fixed your comments!

**Review by elamperti — APPROVED** [review]

**Comment by elamperti** [comment]
Thank you!

**Pull request merged**